app/timetables/models.py

```diff
@@ -249,3 +249,13 @@ class Serving(TimestampMixin):
 
     def __str__(self):
         return '{} served on {}'.format(self.menu_item, self.date_served)
+
+
+class TimetableAutoUpdate(models.Model):
+    """Model to track calls to the timetable auto-update script."""
+
+    timetable = models.ForeignKey(Timetable, on_delete=models.CASCADE)
+    date = models.DateTimeField()
```

Comment:
I think it's desirable to have `default=timezone.now`, from django.utils.timezone.now(), in the DateTimeField.